src/video/windows/SDL_windowsmouse.c

```diff
@@ -84,83 +84,87 @@ static SDL_Cursor *WIN_CreateDefaultCursor()
     return cursor;
 }
 
+static void GenerateBitmapMaskFromAlpha(int width, int height, void *colorBits, void *maskBits)
+{
+    int x, y;
+    const Uint8 *src = colorBits;
+    Uint8 *dst = maskBits;
+    int srcStride = GDI_WIDTHBYTES(width * 32);
+    int dstStride = GDI_WIDTHBYTES(width * 1);
+
+    static const unsigned char masks[] = { 0x80, 0x40, 0x20, 0x10, 0x8, 0x4, 0x2, 0x1 };
```

Comment:
Can you please reformat this with SDL style?
```c
if (x) {
    if (y) {
    }
}
```

Reply:
done :)